debian/control

```diff
@@ -10,8 +10,8 @@ X-Python-Version: >= 2.6
 Package: obs-service-tar-scm
 Architecture: all
 Provides: obs-service-obs-scm, obs-service-tar
-Depends: ${misc:Depends}, ${python:Depends}, bzr, git, subversion, cpio, python-dateutil, python-yaml, locales-all
+Depends: ${misc:Depends}, ${python:Depends}, git, subversion, cpio, python-dateutil, python-yaml, locales-all
 Recommends: mercurial, git-buildpackage, git-lfs, python-keyring, python-keyrings.alt
 Description: An OBS source service: fetches SCM tarballs
  This is a source service for openSUSE Build Service.
- It supports downloading from svn, git, hg and bzr repositories.
+ It supports downloading from svn, git, hg repositories.
```

Comment:
"It supports downloading from svn, git and hg repositories."

Reply:
adjusted